build/LegacyPackages/14.0/Microsoft.Build.Conversion.Core/Microsoft.Build.Conversion.Core.nuspec

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata minClientVersion="2.8.1">
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$id$</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=624683</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <description>This package contains the $id$ assembly which contains logic for converting projects.</description>
+    <tags>MSBuild</tags>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <dependencies>
+      <group targetFramework="netstandard1.3">
+        <dependency id="Microsoft.Build" version="[$version$]" />
```

Comment:
This seemed to break the dependency resolution for netstandard1.3 so I'm going to leave it as is.

---
In reply to: [75924775](https://github.com/Microsoft/msbuild/pull/929#discussion_r75924775) [](ancestors = 75924775)